### PR TITLE
Added an edit flag that will force noedit to be canceled. Closes #130

### LINKED ISSUE #130: add `--edit` flag to force build files editing even if NoEdit set in the config
Seems to be "no" after the see build files choice appears but othwerwise the default is "yes". Might be some reasoning behind this that I'm not aware of.

### PR DESCRIPTION
**Pull request by arivarton**
Simple solution but it works.

### CODE REVIEW

File: pikaur/install_cli.py [hunk 1]
```diff
@@ -626,7 +626,7 @@ def ask_about_package_replacements(self) -> None:
 
     def ask_to_edit_file(self, filename: str, package_build: PackageBuild) -> bool:
         noedit = self.args.noedit or PikaurConfig().build.get('NoEdit')
```
Comment (actionless) [inline pikaur/install_cli.py]: please modify this line instead of the next one

File: pikaur/install_cli.py [hunk 2]
```diff
@@ -625,7 +625,7 @@ def ask_about_package_replacements(self) -> None:
                         self.discard_aur_package(installed_pkg_name)
 
     def ask_to_edit_file(self, filename: str, package_build: PackageBuild) -> bool:
-        noedit = self.args.noedit or PikaurConfig().build.get('NoEdit')
+        noedit = not self.args.edit or self.args.noedit or PikaurConfig().build.get('NoEdit')
```
Comment (actionless) [inline pikaur/install_cli.py]: shouldn't it be `not self.args.edit and (self.args.noedit or PikaurConfig().build.get('NoEdit'))` ?

Comment (arivarton) [reply pikaur/install_cli.py]: Got a little confused. Apparently you need parantheses like this ((noedit1 or noedit2) and not edit) for it to work as expected. Pushing new commit now.

Comment (actionless) [reply pikaur/install_cli.py]: > Apparently you need parantheses for it to work as expected

could you give some examples please?

**Comment by actionless** [comment]
thanks a lot!

**Pull request merged**